### ISSUE TITLE
Sex estimation from the diameter of the radial head?

### ISSUE THREAD

**Issue by LucieBiehlerGomez** [spawaskar-cora/cora-docs#630]
Among morphometric methods for sex estimation, we commonly use the diameter of the humeral, femoral and radial heads. While the first two are already proposed, would it be okay to add the maximum diameter of the radial head to CoRa? 

Here is the reference we use: Berrizbeitia, E., "Sex Determination with the Head of the Radius," Journal of Forensic Sciences, Vol. 34, No. 5, 0, pp. 1206-1213, https://doi.org/10.1520/JFS12754J. ISSN 0022-1198

I don't have the article with me and somehow I couldn't find it, but other references are fine if you prefer.
Basically, the method uses numerical thresholds to assess whether the individual is female, male or if the sex cannot be determined: if the maximum diameter is inferior or equal to 21mm, the individual is female and if it is superior or equal to 23mm, the individual is male. Intermediate measurements are not discriminant enough for a reliable sex assessment.

**Comment by SachinPawaskarUNO** [comment]
@LucieBiehlerGomez I will get this added to CoRA.

**Comment by fedamann** [comment]
@SachinPawaskarUNO Rad_04 in measurements is the associated measurement for the method threshold.  Similarly, the humerus and femur have corresponding measurements (ie, head diameters).  The method will need to be added that then return the result (e.g., male, female, indeterminate) based on the data pulled from the measurement record  that places it among the static thresholds reported in the literature.  Sorry if you are already tracking this...

**Comment by SachinPawaskarUNO** [comment]
@fedamann Thank you, Yes I'm tracking this, we spoke about this earlier today before you joined. @Mirkomattia and @LucieBiehlerGomez are working on a spreadsheet of all stature and sex methods based on the above & other measurements. They plan on sharing that with us in the next couple weeks.
